ts/src/ascendex.ts

```diff
@@ -670,6 +670,7 @@ export default class ascendex extends Exchange {
                         'max': this.safeNumber (market, 'maxNotional'),
                     },
                 },
+                'created': this.safeString (market, 'tradingStartTime'),
```

Comment:
@ttodua string? not consistent with the OKX type 
![image](https://github.com/ccxt/ccxt/assets/43336371/630e96b1-51ba-42c6-9b1d-8a160100bd54)

Reply:
uh oh, mistake

Reply:
fixed